tools/sgtrivy/tools.go

```diff
@@ -0,0 +1,99 @@
+package sgtrivy
+
+import (
+	"context"
+	_ "embed"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+//go:embed trivyignore
+var DefaultConfig []byte
+
+const (
+	version = "0.43.1"
+	name    = "trivy"
+)
+
+func defaultConfigPath() string {
+	return sg.FromToolsDir(name, ".trivyignore")
+}
+
+// CheckTerraformCommand checks terraform configuration on the given dir
+// for any known security misconfigurations.
+func CheckTerraformCommand(ctx context.Context, dir string) *exec.Cmd {
```

Comment:
IMO since this helper only adds a few arguments, might be better to inline this at the call sites to make it more clear how the command is invoked?

Reply:
Sure. It was mostly to make the transition from tfsec easier

Reply:
I've moved the .tryviignore stuff in here now so keeping it

Reply:
Nice!